harvard-imperial-college-london.csl

```diff
@@ -36,7 +39,7 @@
   <macro name="author-short">
     <names variable="author">
       <name form="short" and="symbol" delimiter=", " delimiter-precedes-last="never"/>
-      <et-al font-style="italic"/>
+      <et-al font-style="normal" font-variant="normal"/>
```

Comment:
you can just delete this line; it's only needed when et-al takes additional formatting.

Reply:
deleted